pandas/core/series.py

```diff
@@ -5745,7 +5745,51 @@ def to_period(self, freq: str | None = None, copy: bool | None = None) -> Series
     _info_axis_name: Literal["index"] = "index"
 
     index = properties.AxisProperty(
-        axis=0, doc="The index (axis labels) of the Series."
+        axis=0,
+        doc="""
+        The index (axis labels) of the Series.
+            Get the index (row labels) of the Series.
```

Comment:
https://github.com/pandas-dev/pandas/actions/runs/4608936785/jobs/8145376998?pr=51842#step:8:22

The error states that the summary, i.e. the first block, should only be a single line. So you need to put a blank line between these two lines.

Plus, although no CI job currently complains here, I don't see a reason to indent everything below the first line. Could you dedent by 4 spaces as I did in the suggestion below?

```suggestion
        The index (axis labels) of the Series.
        
        Get the index (row labels) of the Series.
```